### PR TITLE
Minor Fixes in Series methods docs

### PR DESCRIPTION
**Pull request by akharche**

### CODE REVIEW

File: sdc/datatypes/hpat_pandas_series_functions.py [hunk 1]
```diff
@@ -2222,8 +2222,7 @@ def hpat_pandas_series_corr(self, other, method='pearson', min_periods=None):
 
     Limitations
     -----------
-    - 'method' parameter accepts only 'pearson' value. Other values are not supported
-    - Unsupported mixed numeric and string data
+    - Parrameter ``method`` is supported only with default value ``pearson``.
```
Comment (1e-to) [inline sdc/datatypes/hpat_pandas_series_functions.py]: ``'pearson'``

**Comment by 1e-to** [comment]
Also any fixes already exist in PR:
https://github.com/IntelPython/sdc/pull/770

File: sdc/datatypes/hpat_pandas_series_functions.py (hunk 1 above)
Comment (1e-to) [reply sdc/datatypes/hpat_pandas_series_functions.py]: I mean ``'pearson'`` because its str

**Comment by Rubtsowa** [comment]
Also some of these changes are in #780

**Comment by Rubtsowa** [comment]
Also #778

**Comment by akharche** [comment]
It is fully covered by other Pull Requests

**Pull request closed without merging**